server/lib/nl/fulfillment/triple.py

```diff
@@ -37,15 +38,17 @@
 # TODO: revisit how titles should be displayed
 def _get_title(entities: List[Entity], prop_expression: str) -> str:
   entity_str = ', '.join([e.name or e.dcid for e in entities])
-  if prop_expression in OUT_PROP_DISPLAY_NAME or prop_expression.startswith(
-      _OUT_ARROW):
-    prop_display_name = OUT_PROP_DISPLAY_NAME.get(
-        prop_expression, prop_expression[len(_OUT_ARROW):])
-    title_format_str = _OUT_TITLE
-  else:
+  override_title_format = current_app.config['NL_PROP_TITLES'].get(
```

Comment:
nit:  use tmp var for `current_app.config[...]` for readability?

Reply:
done